excel.go

```diff
@@ -225,6 +230,32 @@ func copyCell(f *excelize.File, automatedSheetName string, srcX, srcY, destX, de
 	return nil
 }
 
+func sortRows(f *excelize.File, input *Input, sheetname string) error {
+	rows, err := f.GetRows(sheetname)
+	if err != nil {
+		return err
+	}
+
+	sort.Slice(rows[input.RowOffset:], func(i, j int) bool {
+		return rows[input.RowOffset+i][ColUser] < rows[input.RowOffset+j][ColUser]
+	})
+
+	// write sorted rows to automatedSheet
+	for idx, r := range rows[input.RowOffset:] {
+		cellName := fmt.Sprintf("A%d", 2+idx)
+		err = f.SetSheetRow(sheetname, cellName, &[]string{r[ColUser], r[ColPassword], r[ColPrevious], r[ColTimestamp]})
```

Comment:
`ColUser` etc are "Column" constants. They are not integer indexes and it's just a coincidence that they have these values, isn't it?

How about 
```
err = f.SetSheetRow(sheetname, cellName, r)
```

Reply:
Yes. I asked a question here (don't know where it is!) suggesting that I use indices 0, 1, 2, 3 directly (not the const values. `&r` is the correct way to go. Done.